src/renderer/components/cluster-manager/bottom-bar.tsx

```diff
@@ -5,14 +5,47 @@ import { observer } from "mobx-react";
 import { Icon } from "../icon";
 import { WorkspaceMenu } from "../+workspaces/workspace-menu";
 import { workspaceStore } from "../../../common/workspace-store";
-import { statusBarRegistry } from "../../../extensions/registries";
+import { StatusBarRegistration, statusBarRegistry } from "../../../extensions/registries";
+import _ from "lodash";
 
 @observer
 export class BottomBar extends React.Component {
+  renderRegistrated(registration: StatusBarRegistration) {
```

Comment:
registrated? There are some obscure definitions for it. What about `renderRegisteredItem`?